index.md

```diff
@@ -26,7 +26,7 @@ Our theme focuses on how we can **Do It With Others (DIWO)** as opposed to just
 
 ### Updates
 
-Want to receive updates as we announce more details?
+Want to receive updates as we announce livestream details and more?
```

Comment:
`...announce the conference live stream`?
I think _live stream_ is two words 🤔

Reply:
🤷‍♀️ i think both "livestream" and "livestream" are used, we just need to pick one and be consistent. What is our preference?